packages/modules/vehicles/vwid/libvwid.py

```diff
@@ -68,8 +68,8 @@ def password_form(self, text):
 
             return (form, action)
 
-        except KeyError:
-            self.log.error("Missing fields in response from VW API")
+        except KeyError as exc:
+            self.log.error(f"Missing fields in response from VW API ({exc})")
```

Comment:
```suggestion
        except KeyError:
            self.log.exception("Missing fields in response from VW API")
```
So bekommt man automatisch die Exception geloggt.